src/models/user.js

```diff
@@ -102,5 +105,9 @@ export default (sequelize, DataTypes) => {
       onDelete: 'CASCADE',
     });
   };
+
+  User.hashPassword = password => bcrypt.hashSync(password, bcrypt.genSaltSync(10));
```

Comment:
You can remove this. @Raymond-Osy has proper implementation.

Reply:
You can relax on this. If I merge @Raymond-Osy work first then you gonna remove it.